bin/p2-balancer/main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/hashicorp/consul/api"
+	"github.com/square/p2/pkg/balancer"
+	"github.com/square/p2/pkg/config"
+	"github.com/square/p2/pkg/logging"
+)
+
+// serpro (service proxy) is a simple TCP proxy for services registered
+// in the Consul registry.
+
+func main() {
+	cfg, err := config.LoadFromEnvironment()
+	logger := logging.DefaultLogger
+
+	if err != nil {
```

Comment:
A lot of the clarifying details about the errors will be included in the "err" field, but I will revise some of these to be clearer.